cmd/datamon/cmd/cli_workshop_test.go

```diff
@@ -0,0 +1,494 @@
+		files[name].downloaded = true
+		assert.Equal(t, files[name].size, int(info.Size()))
+		b, erf := ioutil.ReadFile(pth)
+		require.NoError(t, erf)
+		assert.EqualValues(t, files[name].data, b)
+		return nil
+	})
+	require.NoError(t, err)
+	for name, file := range files {
+		assert.Truef(t, file.downloaded, "expected file %s to be downloaded", name)
+	}
+
+	// downloading some files
+	pathToSomeDownload, _ := ioutil.TempDir("", "mmd-")
+	defer os.RemoveAll(pathToSomeDownload)
+
+	var picked string
+	for n := range files {
+		picked = n
+		break
+	}
+
+	cmd, _ = testCommand(t, false, false, targetBinary,
+		"bundle",
+		"download",
+		"--repo", repo,
+		"--label", label,
+		"--destination", pathToSomeDownload,
+		"--name-filter", picked[:8]+".+",
+	)
+	output, err = cmd.CombinedOutput()
+	require.NoErrorf(t, err, "unexpected error, got: %s", string(output))
+
+	err = filepath.Walk(pathToSomeDownload, func(pth string, info os.FileInfo, _ error) error {
+		rel, _ := filepath.Rel(pathToSomeDownload, pth)
+		if info.IsDir() || model.IsGeneratedFile(rel) {
+			return nil
+		}
+		name := filepath.Base(pth)
+		require.Equal(t, picked, name)
+		assert.Equal(t, files[name].size, int(info.Size()))
+		b, erf := ioutil.ReadFile(pth)
+		require.NoError(t, erf)
+		assert.EqualValues(t, files[name].data, b)
+		return nil
+	})
+	require.NoError(t, err)
+
+	// no test here for web (see pkg/web)
+	// no test here for upgrade (see self_upgrade_test.go)
+}
+
+func makeTestWorkshopBundle(t testing.TB, pathToData string) map[string]*testFile {
+	files := make(map[string]*testFile, 10)
+	for i := 0; i < 9; i++ {
+		name := internal.RandStringBytesMaskImprSrc(10)
+		data := internal.RandBytesMaskImprSrc(100)
+		err := ioutil.WriteFile(filepath.Join(pathToData, name), data, 0644)
+		require.NoError(t, err)
+		files[name] = &testFile{data: data, size: 100}
+	}
+	name := internal.RandStringBytesMaskImprSrc(10)
+	data := []byte{} // empty file
+	err := ioutil.WriteFile(filepath.Join(pathToData, name), data, 0644)
+	require.NoError(t, err)
+	files[name] = &testFile{data: data, size: 0}
+	return files
+}
+
+func TestWorkshopBrewInstall(t *testing.T) {
+	if runtime.GOOS == "linux" { // TODO: don't know where brew installs things on OSX
+		home, _ := os.UserHomeDir()
+		cmdPath := strings.Join([]string{
+			filepath.Join("/", "home", "linuxbrew", ".linuxbrew", "bin"),
+			filepath.Join(home, ".linuxbrew", "bin"),
+			os.Getenv("PATH"),
+		}, ":")
+		os.Setenv("PATH", cmdPath)
+	}
+
+	// brew install (linux flavor)
+	cmd := exec.Command("brew", "--version")
+	if output, err := cmd.Output(); err != nil {
+		t.Logf("brew not installed. Skipping test")
+		t.SkipNow()
+	} else {
+		t.Logf("using brew: %s", string(output))
+	}
+
+	output, err := exec.Command("brew", "tap", "oneconcern/datamon").CombinedOutput()
+	require.NoErrorf(t, err, "could not run brew tap: %v:\n%s", err, string(output))
+
+	output, err = exec.Command("brew", "install", "datamon").CombinedOutput()
```

Comment:
@kerneltime @ransomw1c you may notice here that the install here works as initially advertised and not using brew install datamon/datamon. Perhaps this is a specificity of linuxbrew.